lsp-rust.el

```diff
@@ -426,6 +428,12 @@ PARAMS progress report notification data."
       (find-file filename)
       (goto-char (lsp--position-to-point position)))))
 
+(defun lsp-rust-select-and-apply-source-change-command (p)
+  (let* ((options (-> p (ht-get "arguments") (lsp-seq-first)))
+         (chosen-option (lsp--completing-read "Select option:" options
+                                              (-lambda ((&hash "label")) label))))
+    (lsp-rust-apply-source-change-command chosen-option)))
```

Comment:
this needs to be `lsp-rust-apply-source-change`

Reply:
Thanks ;)